testing/test/t/curity.t

```diff
@@ -324,4 +345,63 @@ GET /t
 content-type: application/json
 
 --- response_body_like chomp
-{"code":"server_error","message":"Problem encountered processing the request"}
\ No newline at end of file
+{"code":"server_error","message":"Problem encountered processing the request"}
+
+=== TEST 11: A large JWT can be processed with the correct proxy buffer configuration
```

Comment:
The first test verifies that a large JWT can be processed with caching disabled.